components/dot_menu/dot_menu.scss

```diff
@@ -0,0 +1,15 @@
+.dotmenu-dropdown-menu {
```

Comment:
I would prefer to name this `dropdown-menu__dotmenu` for a simple reason: We already have the class `dropdown-menu` and this i just a variation of that.